airflow/providers/google/cloud/transfers/salesforce_to_gcs.py

```diff
@@ -0,0 +1,106 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import csv
+import json
+import tempfile
+from typing import Dict, Optional
+
+from airflow.models import BaseOperator
+from airflow.providers.google.cloud.hooks.gcs import GCSHook
+from airflow.providers.salesforce.hooks.salesforce import SalesforceHook
+
+
+class SalesforceToGcsOperator(BaseOperator):
+    """
+    Submits Salesforce query and uploads results to Google Cloud Storage
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:SalesforceToGcsOperator`
+
+    :param query: The query to make to Salesforce.
+        :type query: str
```

Comment:
```suggestion
    :param query: The query to make to Salesforce.
    :type query: str
```
I'm thinking that this would give error if not on the same line with the param?

Reply:
Thanks, fixed formatting for all parameters in this doc string.